numba/core/types/containers.py

```diff
@@ -766,13 +765,22 @@ class LiteralStrKeyDict(Literal, ConstSized, Hashable):
     namedtuple with dict semantics).
     """
 
+    class FakeNamedTuple(object):
```

Comment:
Shouldn't this inherit from `Sequence`? The old type for `tuple_ty` is a `namedtuple` and it is not hidden (no `_` prefix), so users might have code expecting it has tuple-like API.

Reply:
I've updated this to inherit from `collections.abc.Sequence` and to implement `__len__` and `__getitem__` by delegating to the `._fields` member.